src/cmd/reconcile.go

```diff
@@ -14,15 +14,19 @@ var reconcileResyncInterval int
 
 var reconcileCmd = &cobra.Command{
 	Use:   "reconcile",
-	Short: "Run in the foreground as a kubernetes controller to reconcile data with service entries in OpsLevel",
+	Short: "Run as a kubernetes controller to reconcile data in OpsLevel",
 	Long:  `Run in the foreground as a kubernetes controller to reconcile data with service entries in OpsLevel`,
 	Run: func(cmd *cobra.Command, args []string) {
+		var (
+			err    error
+			resync time.Duration = time.Hour * time.Duration(reconcileResyncInterval)
+		)
+		cobra.CheckErr(err)
```

Comment:
why are we running `checkErr` here it has not been set yet?

Reply:
:))